### ISSUE TITLE
On final page, not all image cards loaded 

### ISSUE THREAD

**Issue by laurafulton** [cmuvis/mlint#20]
Try to understand why last card did not load.
<img width="1348" alt="Screen Shot 2019-10-13 at 3 59 35 PM" src="https://user-images.githubusercontent.com/9020517/66723065-de54c500-ede2-11e9-810e-5f0257f6fa76.png">

**Comment by adamperer** [comment]
Decide if this is the desired design or not, before fixing.

**Comment by laurafulton** [comment]
another example
<img width="1101" alt="error in final screen" src="https://user-images.githubusercontent.com/9020517/67613370-a6eafe80-f77a-11e9-8364-6b2a8b14f7a4.PNG">

**Comment by yzd1998111** [comment]
The issue has already been resolved after I fixed the previous image loading issue. 
I did some testing. When there were some images not loading up. The final score image is also broken. I achieve this by manually breaking all the images for animals. Then I got something like this.
<img width="1419" alt="Screen Shot 2019-11-02 at 1 31 05 PM" src="https://user-images.githubusercontent.com/17274134/68074757-25483180-fd75-11e9-8b03-21cca9326a67.png">

Otherwise, it always showed up correctly. (I tested this four times.)
